### ISSUE TITLE
7.33 Phylactery + Spirit Breaker vs Nyx

### ISSUE THREAD

**Issue by daniellogerstedt** [ValveSoftware/Dota2-Gameplay#8642]
### Item name

Phylactery

### Description

There is a really bizarre interaction that occurs specifically when you have Phylactery on Spirit Breaker and then charge Nyx while he has Spiked Carapace up. 

When charge is used on Nyx as or while spiked carapace is active SB will get stunned instantly because of the phylactery hit then it appears SB is running in place for a while and charge remains on cooldown (at 0 seconds if the charge distance is great enough) but the charge cannot be cancelled as though the game doesn't recognize that it exists. Despite the charge "not existing" while also "existing" at the same time after a while the Nyx gets hit with a greater bash as though you had landed the charge successfully even though SB remains where ever they've moved to since the charge was initiated. 

This can be recreated by opening a demo lobby with SB and Nyx, get Phylactery on SB and then use spiked carapace and charge the Nyx with SB. It repeats the situation every time without fail. 

SB Telekinetic bashes while fun are a bit obnoxious if you do a long charge as someone pops carapace and are stuck in the animation for a lengthy amount of time. It is great that Phylactery works with charge though it gives SB more viability as a support for helping team mates quickly.

The included match has the bug occur a few seconds after the timestamp (13:20) where SB (in Dire Fountain) charges Nyx (in Radiant Mid Jungle near T1 Mid) if you watch SB gets locked in the charge animation after the carapace stun wears off and the cooldown of charge goes to 0 but then stays there until Nyx is eventually hit with a greater bash at around 14:05

### Example Match ID and Timestamp

7124804928 - 13:20 - 14:05

**Comment by 6222780106** [comment]
Yes, when SB casts Charge of Darkness to Nyx, Charge of Darkness will immediately be on cooldown, but the effect of Charge of Darkness will be applied after Phylactery. In other words, Phylactery will always deal a separate damage to Nyx before the Charge of Darkness takes effect. With Spiked Carapace, Phylactery's damage will induce stun to SB. After that, SB will receive the buff from Charge of Darkness that will disappear immediately because SB is currently stunned.
